services/user/Chainmail/ui/src/components/markdown-editor.tsx

```diff
@@ -33,6 +26,11 @@ import {
 
 // import { MathBlock, MermaidDiagram } from "./editor";
 // import { MathBlock } from "./editor";
```

Comment:
I assume all these commented imported can be deleted? Couple commented lines below as well

Reply:
They can be, but I'm keeping them around now for reimplementation of future editor features/markdown stuff.